synapse/handlers/room_summary.py

```diff
@@ -476,11 +498,16 @@ async def _summarize_local_room(
                 Otherwise, all children are returned.
             include_children:
                 Whether to include the events of any children.
+            admin_skip_room_visibility_check: Whether to skip checking if the room can be accessed by the requester,
+                used when the requester is a server admin
```

Comment:
```suggestion
            admin_skip_room_visibility_check: Whether to skip checking if the room can be accessed by the requester,
                used for the admin endpoints.
```

This seems more accurate. As someone could be an admin but we don't want this set for the client endpoint.

(applies to other places this docstring exists)